src/Domain/TagsTimeDependent.hpp

```diff
@@ -160,5 +160,11 @@ struct InertialMeshVelocityCompute : MeshVelocity<Dim, Frame::Inertial>,
 
   using argument_tags = tmpl::list<CoordinatesMeshVelocityAndJacobians<Dim>>;
 };
+
+/// The divergence of the mesh velocity
+struct DivMeshVelocity : db::SimpleTag {
+  using type = boost::optional<Scalar<DataVector>>;
+  static std::string name() noexcept { return "div(MeshVelocity)"; }
```

Comment:
I think you're overlooking the `boost::optional`, which is not part of `Tags::div` (I thought about having this just be a `Tags::div` but then realized that doesn't work because of the different types)

I think we currently ignore `boost::optional` in the name. Since the names will (hopefully) eventually be used for retrieving things out of the DataBox for observing at runtime, it'll need to be dealt with there, and so probably having it just be `div(MeshVelocity)` would be clearer. However, I'm not attached to this :)

Reply:
You are correct, I did overlook the `boost::optional` 🙁. I am satisfied with how this is currently structured - thanks for clarifying this!

Reply:
Should I add a comment into the code?